### PR TITLE
LPS-131420 Add order by due date column

### PR DESCRIPTION
**Pull request by jotascript**
Story: [LPS-131420](https://issues.liferay.com/browse/LPS-130167)

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/InstanceListPageTable.es.js [hunk 1]
```diff
@@ -25,7 +26,10 @@ const Table = ({items, totalCount}) => {
 					</ClayTable.Cell>
 
 					<ClayTable.Cell headingCell style={{width: '14%'}}>
-						{Liferay.Language.get('due-date')}
+						<ListHeadItem
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/InstanceListPageTable.es.js:29]: Hey @jotascript, rename it to `dueDate` instead and apply it to other places, if any. Thx Bro

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:help

**Review by ramonmsp — APPROVED** [review]

**Comment by rafaprax** [comment]
ci:test:workflow-metrics

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**